src/foam/u2/crunch/CrunchController.js

```diff
@@ -264,5 +264,11 @@ foam.CLASS({
 
       return p;
     },
+
+    function getCapableWizard(capable) {
```

Comment:
Now that I think about it `getWizardletsFromCapable` is a better name, and maybe there should be a comment mentioning that this is specifically for CapableView.

Reply:
Updated. Thank you :-)